### DISCUSSION TITLE
Documentation for gRPC connector in LB4

### DISCUSSION THREAD

**Discussion by ssraj326** [loopbackio/loopback-next#7619] [Q&A] [answered]
Im trying to setup gRpc client using service-proxy. But I couldnt find much help. I use Loopback4 in our project.

this link is also not yet updated.
https://loopback.io/doc/en/lb4/gRPC-connector.html

would be great if we have some documentation for the grpc connector.
Can  someone help me to setup the gRPC client in LB4.

**Comment by marioestradarosa** [comment, marked as answer]
@ssraj326 as @dhmlau mentioned on the slack channel, that link should be pointing the loopback-connector-grpc repository located [here](https://github.com/loopbackio/loopback-connector-grpc).

At the same time, I have added in my personal repository a gRPC client example with a tutorial in the readme file. 

https://github.com/marioestradarosa/loopback-grpc-client-example
